client.js

```diff
@@ -126,16 +140,16 @@ function getState() {
 function recorderProcess(e) {
     var samp = e.inputBuffer.getChannelData(0)
     state = getState()
-    if(state == 'gray') return
+    if(state === 'gray') return
     for(var i = 0; i < e.inputBuffer.length; i++) {
-        if(Math.abs(samp[i]) > 0.1) {
-            if(state == 'recording') {
+        if(Math.abs(samp[i]) > 0.4) { // volum audio
```

Comment:
cred că-i prea puțin 40% volum. 10% era mai okay. Trebuie să poți transmite și chestii gen "pfffff" sau "nț" și ălea au volum destul de mic. Sau ai sensibilitatea microfonului prea mică (gen volum puternic ajunge doar pe la 50%). Sau dacă un cuvânt începe cu o consoană mută (t, p, c) s-ar putea să nu se prindă că vorbești decât când ajungi la prima vocală. Eventual dacă e să punem volumul de detecție la 40% ar trebui să trimitem și vreo 50ms din urmă.